.buildkite/pipeline.yml

```diff
@@ -72,7 +72,7 @@ steps:
       else CONTRACT_DOC_STEP="contract doc release";
       fi
     - buildkite-agent artifact download Holdings-contract.md tmp/
-        --step "$CONTRACT_DOC_STEP"
+        --step "$$CONTRACT_DOC_STEP"
```

Comment:
How does it work? What syntax is this?

Reply:
Why two dollars?

Reply:
Apparently, buildkite eats the first dollar and the second dollar eats shell

Reply:
Bon appetite to both of them.